docs/sources/flow/_index.md

```diff
@@ -9,11 +9,13 @@ description: Grafana Agent Flow is a component-based revision of Grafana Agent w
   a focus on ease-of-use, debuggability, and adaptability
 title: Flow mode
 weight: 400
+cascade:
+  PRODUCT_NAME: Grafana Agent Flow
 ---
 
-# Flow mode
+# {{< param "PRODUCT_NAME" >}}
```

Comment:
`title:` and `# {{< param "PRODUCT_NAME" >}}` must match, so I imagine this would need to be:
* `title: {{< param "PRODUCT_NAME" >}} mode`
* `# {{< param "PRODUCT_NAME" >}} mode`

Reply:
Hmm testing, and coascade vars can't be used in the metadata section (makes sense). So, either the few that are like this will need to be manually updated, or the H1 on pages that do this need to be reworked to remove any direct reference to the product name. In the Agent docs, there's only a couple of topics that do this.